src/const.h

```diff
@@ -681,6 +681,11 @@ enum ReloadTypes_t : uint8_t
 	RELOAD_TYPE_WEAPONS,
 };
 
+enum SquareColor_t : uint8_t 
```

Comment:
this already exists, also you don't need to send it to lua, since the same colors are used as the text.